addons/payment/data/portal_entry_data.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="portal_payment_orders" model="portal.entry">
```

Comment:
There are no orders in `payment`. `payment_methods_portal_entry` would be a more fitting XMLID.
```suggestion
    <record id="payment_methods_portal_entry" model="portal.entry">
```